flair/trie/replication.go

```diff
@@ -84,9 +84,9 @@ func (r *Replicator) SequentialAck(key string, f ReplicatedAckFunc) error {
 		}
 		merr = multierror.Append(merr, err)
 		if i < r.Replicas-1 {
-			log.Debug("Error reading from replica for %s: %s. Will retry on next replica.", key, err)
+			log.Error("Error reading from replica for %s: %s. Will retry on next replica.", key, err)
```

Comment:
nit: line 87, 89, 131 could have the `err` verb as  `%w`

Reply:
Good spot. I have changed these to %v. Githubs UI isn't great here but you can see the changes if you hit Files changed.